explore/src/main/scala/explore/itc/ITCResultsCache.scala

```diff
@@ -57,7 +55,7 @@ final case class ItcResultsCache(
 object ItcResultsCache {
 
   def enabledRow(row: SpectroscopyModeRow): Boolean =
-    List(Instrument.GmosNorth, Instrument.GmosSouth).contains_(
+    List[Instrument](Instrument.GmosNorth).contains_(
```

Comment:
Why only GmosNorth now?

Reply:
Currently ITC only supports GN and it was being filtered twice, This one is not needed anymore but would need to be restored as we enable more instruments